File: 01-input-output/05-sum-two-numbers/solution.c
#include <stdio.h>


float get_number(char *label);

int main(void){
    char *first = "first";
    float first_number = get_number(first);

    char *second = "second";
    float second_number = get_number(second);

    float sum = first_number + second_number;

    printf("Sum of %f and %f is %f.\n", first_number, second_number, sum);
    return 0;
}


float get_number(char *label){
    printf("Enter %s number : ", label);
    float number;
    scanf("%f", &number);
    return number;
}

File: 01-input-output/05-sum-two-numbers/solution.js
import readline from 'readline/promises';
import { stdin as input, stdout as output } from 'process';

const rl = readline.createInterface({
    input, output
});

let firstNumber = (await rl.question("Enter first number : ")).trim();
let secondNumber = (await rl.question("Enter second number : ")).trim();

firstNumber = parseFloat(firstNumber)
secondNumber = parseFloat(secondNumber)

console.log(`Sum of ${firstNumber} and ${secondNumber} is ${firstNumber + secondNumber}`)
rl.close()